package.json

```diff
@@ -3,9 +3,14 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "axios": "^0.17.1",
```

Comment:
`axios` is used here.

Note that the `fetch` library provided is native to every browser. (Except IE11, SAD!)